katalog/cert-manager/webhook/webhookvalidatingconfig.yml

```diff
@@ -70,24 +72,16 @@ webhooks:
           - UPDATE
         resources:
           - "*/*"
-    # We do not support Kubernetes runtimes < 1.19
     admissionReviewVersions: ["v1"]
     # This webhook only accepts v1 cert-manager resources.
     # Equivalent matchPolicy ensures that non-v1 resource requests are sent to
     # this webhook (after the resources have been converted to v1).
     matchPolicy: Equivalent
+    timeoutSeconds: 30
     failurePolicy: Fail
     sideEffects: None
     clientConfig:
       service:
         name: cert-manager-webhook
-        namespace: "cert-manager"
+        namespace: cert-manager
         path: /validate
-    timeoutSeconds: 10
----
```

Comment:
I don't know if this configMap was added manually and could be removed

@nutellinoit @ralgozino

Reply:
according to the comment I added it because it was included in the manifests generated by upstream, if it is not included in upstream anymore I think we can remove it.